common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -780,6 +780,11 @@
           "Cannot retrieve <objectName> from the ML cache. It is probably because the entry has been evicted."
         ]
       },
+      "CACHE_ITEM_EXCEEDED" : {
+        "message" : [
+          "Cannot <operation> object <name>, since its estimated size <estimatedSize> exceeds the limitation <maximumSize>."
```

Comment:
`...exceeds the maximum allowed size <maximumSize>.` instead of `exceeds the limitation <maximumSize>.`?

Does this use proper size formats?